packages/editor/src/components/properties/SplineNodeEditor.tsx

```diff
@@ -82,15 +83,21 @@ export const SplineNodeEditor: EditorComponentType = (props) => {
                 smallStep={0.01}
                 mediumStep={0.1}
                 largeStep={1}
-                onChange={commitProperty(SplineComponent, `elements.${index}.position` as any)}
+                onChange={(position) => {
+                  const changePosition = commitProperty(SplineComponent, `elements.${index}.position` as any)
```

Comment:
should call as `commitProperty(SplineComponent, `elements.${index}.position` as any)(new Vector3(position.x, position.y, position.z))`

Reply:
yup , on it